### PR TITLE
Upgrade ANO to DOCA 2.8

### PR DESCRIPTION
**Pull request by e-ago**

### CODE REVIEW

File: operators/advanced_network/Dockerfile [hunk 1]
```diff
@@ -24,13 +24,20 @@ ARG BUILDARCH
...
 WORKDIR /opt
 
-RUN    eval `dpkg-architecture` \
-        && wget https://www.mellanox.com/downloads/DOCA/DOCA_v2.7.0/host/doca-host_2.7.0-204000-24.04-ubuntu2204_${BUILDARCH}.deb -O doca-host.deb \
-        && ls -lh && apt install ./doca-host.deb \
-        && apt update \
-        && apt install -y doca-all \
-        && apt install -y doca-gpu doca-gpu-dev
+ARG DEBIAN_FRONTEND=noninteractive
+ARG DOCA_REPO_LINK=https://linux.mellanox.com/public/repo/doca/2.8.0/ubuntu22.04/x86_64
```
Comment (cliffburdick) [inline operators/advanced_network/Dockerfile:32]: This looks like it removes BUILDARCH

Comment (e-ago) [reply operators/advanced_network/Dockerfile:32]: because BUILDARCH says amd64 but the link is at x86_64

**Comment by e-ago** [comment]
@cliffburdick @jjomier if we can merge it today so rmax folks can submit their MR

File: operators/advanced_network/Dockerfile (hunk 1 above)
Comment (cliffburdick) [reply operators/advanced_network/Dockerfile:32]: Should we parameterize on that so we don't need two dockerfiles?

**Review by cliffburdick — APPROVED** [review]

**Comment by cliffburdick** [comment]
@e-ago I don't have permission to merge

**Pull request merged**